packages/sdk-components-react/src/xml-time.tsx

```diff
@@ -0,0 +1,62 @@
+import { forwardRef, useContext, type ElementRef } from "react";
+import { ReactSdkContext } from "@webstudio-is/react-sdk/runtime";
+
+const DEFAULT_DATE_STYLE = "short";
+const INITIAL_DATE_STRING = "dateTime attribute is not set";
+const INVALID_DATE_STRING = "";
+
+type XmlTimeProps = {
+  dateStyle: "long" | "short";
+  datetime: string;
+};
+
+const parseDate = (datetimeString: string) => {
```

Comment:
couldn't we reuse the logic from the other time component? it feels like this stuff needs lots of tests

Reply:
Something with build:args on reuse. It calcs exports as props. So I would spend tooo many time on reuse. Here is just few lines

Reply:
so we can't import any modules from that package? that would be crazy @TrySound do you know what's going on?

Reply:
We can't export something not related to component i.e. props component itself otherwise type builder assumes this as props.
I just don't want waste time to find how to fix/improve it. As this code is so small that does not need any reuse.

Reply:
I understand the urge and maybe we can merge this, but I think its a big problem in general, not only for this, I don't want us to be fixing this thing after users find a bug without a way to ensure there are no regressions and not having tests for it, working with dates is always a pain ...